includes/specials/SpecialConfirmemail.php

```diff
@@ -123,7 +123,7 @@ function showRequestForm() {
 			if( $user->isEmailConfirmationPending() ) {
 				$out->wrapWikiMsg( "<div class=\"error mw-confirmemail-pending\">\n$1\n</div>", 'confirmemail_pending' );
 				// Wikia change -- only allow one email confirmation attempt per hour
-				if (strtotime($user->mEmailTokenExpires) - strtotime("+6 days 23 hours") > 0) return;
+				if (strtotime($user->getEmailTokenExpires()) - strtotime("+6 days 23 hours") > 0) return;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)